### PR TITLE
Download built jar from mvn instead of building with ant

### PR DESCRIPTION
**Pull request by MatrixFrog**

### CODE REVIEW

**Comment by MatrixFrog** [comment]
We could also *build* using the `mvn` command line but then getting the version number right is trickier. Since we will have already pushed the mvn release by this point, it's simpler to just download the jar.

**Comment by ChadKillingsworth** [comment]
I'm fine with the concept - but skeptical that maven releases will be timely. We could also download from dl.google.com. That's much more reliable.

**Comment by MatrixFrog** [comment]
They will be, because the team has taken them over. In fact, the dl.google.com jar now comes from mvn as well (because ant is going away for us). So the new order of things will be:

1. mvn release
2. dl.google.com
3. npm

**Comment by ChadKillingsworth** [comment]
So the tricky part here is making sure that the submodule commit sha is in sync with the maven release. I'm not sure of a way to add a test for that - but it would be nice to make sure that the build and the externs were from the same point in history.

**Comment by MatrixFrog** [comment]
The compiler commit should be tagged. We could add a test that `cd`s into the `compiler` directory and checks for a tag whose name matches the version in package.json, and then verifies that HEAD and that tag are the same commit.

**Comment by blickly** [comment]
I'd assume it would look something like:
```js
const currentTag = spawn('git tag --points-at HEAD').stdout;
assert(/^v\d{8}/.test(currentTag));
```

**Comment by MatrixFrog** [comment]
Okay, I'll take a stab at it.

File: build_compiler.js [hunk 1]
```diff
@@ -3,24 +3,21 @@
 
 var spawn = require('child_process').spawnSync;
 var ncp = require('ncp');
+var version = require('./package.json').version
 
-var compilerBuild = spawn('ant', ['jar'], {
-  cwd: './compiler',
+var mavenVersion = 'v' + version.split('.')[0];
+var url = 'https://repo1.maven.org/maven2/com/google/javascript/closure-compiler/' + mavenVersion + '/closure-compiler-' + mavenVersion + '.jar';
```
Comment (ChadKillingsworth) [inline build_compiler.js]: Can you wrap this line?

Comment (MatrixFrog) [inline build_compiler.js]: Done.

**Comment by ChadKillingsworth** [comment]
👍

**Pull request merged**